opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -243,6 +243,8 @@ namespace Opm {
                 residual_norms_history_.clear();
                 current_relaxation_ = 1.0;
                 dx_old_ = 0.0;
+                convertInput( iteration, reservoir_state, ebosSimulator_ );
```

Comment:
I agree we should not change that aspect of the Frankenstein right now. The important part about this is how it changes the simulator runs though.